internal/web/templates/pages/settings.html

```diff
@@ -19,7 +19,12 @@ <h2 class="text-reset">Package Repositories</h2>
                     {{ end }}
                   </div>
                   <div class="ms-2 d-flex flex-column">
-                    <span class="fw-semibold">{{ .Name }}</span>
+                    <span class="fw-semibold">
+                      {{ .Name }}
+                      {{ if IsRepoStatusReady . }}
```

Comment:
I think you intended to call your `IsDefaultRepository` function here instead of the `IsRepoStatusReady`.

This can be simplified, because the `PackageRepository` type already has a `IsDefaultRepository` function (see `api/v1alpha1/packagerepository_types.go`, and you can call such functions directly from the template. So you can remove the `IsDefaultRepository` function from the `templateFuncs`, and simply call `.IsDefaultRepository` in the template.

Reply:
Thanks for your suggestion, It's now updated!